bubblewrap.c

```diff
@@ -2677,6 +2685,10 @@ main (int    argc,
     usage (EXIT_FAILURE, stderr);
 
   parse_args (&argc, (const char ***) &argv);
+#if 0
+  if (!specified_explicitly_opt_new_session)
+    warn ("No new terminal session creation behaviour specified. Please specify it explicitly with either --new-session or --no-new-session. Will default to --no-new-session.");
```

Comment:
Now that we have a warning that's wrapped in `#if 0`, I think that might be a sign that it's time to add a `--debug` option, and a `debug()` that is similar to `warn()` but doesn't print anything unless `--debug` was given.